templates/user/comment.html

```diff
@@ -0,0 +1,124 @@
+{% extends "base.html" %}
```

Comment:
this file is somewhat familiar with `comments/list.html`, could we somehow use that file?

Reply:
The structure of `comments/list.html` is fundamentally different than that of what I'm trying to achieve: `comments/list.html` is based on a tree structure showing relationships between comments, while I expect the comments of the `Comments` page to be just a chronologically sorted list.

Also, `comments/list.html` contains other parts, like the guidelines warning and the comment form, which don't make sense to appear on the page.

Reply:
Thanks for the clarify